guides/common/modules/ref_ports-and-firewall-requirements.adoc

```diff
@@ -15,35 +15,41 @@ This includes the base operating system on which {SmartProxyServer} is running.
 
 .Clients of {SmartProxy}
 Hosts which are clients of {SmartProxies}, other than {Project}'s integrated {SmartProxy}, do not need access to {ProjectServer}.
+ifdef::satellite[]
 For more information on {Project} Topology, see {PlanningDocURL}sect-Documentation-Architecture_Guide-Capsule_Networking[{SmartProxy} Networking] in _Planning for {ProjectNameX}_.
+endif::[]
 
 Required ports can change based on your configuration.
 
-ifeval::["{build}" != "foreman-deb"]
+ifeval::satellite[]
 A matrix table of ports is available in the Red{nbsp}Hat Knowledgebase solution https://access.redhat.com/solutions/5627751[Red Hat Satellite List of Network Ports].
 endif::[]
 
 The following tables indicate the destination port and the direction of network traffic:
 
+ifdef::satellite[]
 ifeval::["{mode}" == "connected"]
-
 .Ports for {Project} to Red Hat CDN Communication
 [cols="24%,20%,18%,38%",options="header"]
 |====
 | Port | Protocol | Service | Required For
 | 443 | TCP | HTTPS | Subscription Management Services (access.redhat.com) and connecting to the Red{nbsp}Hat CDN (cdn.redhat.com).
 |====
+endif::[]
+endif::[]
 
-ifeval::["{build}" == "satellite"]
+ifdef::satellite[]
+ifeval::["{mode}" == "connected"]
```

Comment:
Oh is the plan to merge connected and disconnected guides into single content? That would be nice.

Reply:
I think both guides are currently made from the same source!